ddp-workspace/projects/ddp-dsm-ui/src/app/shipping/shipping.component.html

```diff
@@ -310,7 +310,7 @@ <h4>There are no kit requests</h4>
           </td>
           <td [ngClass]="{'highlight_sample': isResearchSample(kitRequest)}">
             <!--            kit type-->
-            {{kitRequest.kitTypeName}}
+            {{kitRequest.kitTypeDisplayName}}
```

Comment:
The checking in here is not needed

Reply:
Will this (not checking) affect legacy participants?